scalekit/core.py

```
@@ -150,30 +153,64 @@ def get_headers(self, headers: Optional[dict] = None) -> dict:
     def grpc_exec(
         self,
         func: WithCall,
         data: TRequest,
-        retry=2,
+        _retry: int = 2,
     ) -> TResponse:
+        if self._token_needs_refresh():
+            try:
+                self.__authenticate_client()
+            except Exception:
+                pass  # token still valid within buffer; let call proceed and handle reactively
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)